src/test/scala-2.11/org/apache/james/gatling/smtp/scenari/NoAuthenticationNoEncryptionBigBodyScenario.scala

```diff
@@ -13,7 +13,7 @@ class NoAuthenticationNoEncryptionBigBodyScenario extends Simulation {
   val myRandom = Random.alphanumeric
 
   def generateMessage() : String =
-    myRandom take (1024 * 1024) mkString
+    myRandom take (1024 * 1024) grouped 200 map (_ append Stream('\n')) flatMap identity mkString
```

Comment:
are you sure `\n` is sufficient? I guess mails are based on `\r\n`
(btw it's a 1% increase of the size of the mail, but as you said me, not sure it's worth caring of it)